lib/sgov/sgov-adapter.spec.ts

```diff
@@ -0,0 +1,117 @@
+import {SgovAdapter} from "./sgov-adapter";
+import {httpFetch} from "../io/fetch/fetch-nodejs";
+import {PimClass} from "../pim/model";
+import {CimAdapter, IriProvider, PrefixIriProvider} from "../cim";
+import {FetchOptions} from "../io/fetch/fetch-api";
+import {CoreResourceReader} from "../core";
+
+let iriProvider: IriProvider;
+let adapter: CimAdapter;
+
+beforeAll(() => {
+  iriProvider = new PrefixIriProvider();
+  adapter = new SgovAdapter("https://slovník.gov.cz/sparql", httpFetch);
+  adapter.setIriProvider(iriProvider);
+});
+
+test("SgovAdapter.search()", async () => {
+  const query = "řidič";
+  const result = await adapter.search(query);
+  expect(result.map(cls => cls.pimHumanLabel?.cs)).toContain("Řidičský průkaz České republiky");
+}, 10 * 60 * 1000);
+
+describe("SgovAdapter.getClass()", () => {
+  test("existing class", async () => {
```

Comment:
I would start the test "names" with capital letter , but it is true that together with describe it is more complicated so it depends on the output probably.